Documentation/ceph-object-multisite.md

```diff
@@ -41,12 +41,45 @@ object-multisite.yaml in the [examples](/cluster/examples/kubernetes/ceph/) dire
 kubectl create -f object-multisite.yaml
 ```
 
-The first zone group created in a realm is the master zone group. The first zone created in a zone group is the master zone.
+The first zone group created in a realm is the master zone group. The first zone created in a zone group is the master zone. When a non-master zone or non-master zone group is created, the zone group or zone is not in the Ceph Radosgw Multisite [Period](https://docs.ceph.com/docs/master/radosgw/multisite/) until an object-store is created in that zone (and zone group).
 
-When one of the multisite CRs (realm, zone group, zone) is deleted the underlying ceph realm/zone group/zone is not deleted. This must be done manually (see next section).
+The zone will create the pools for the object-store(s) that are in the zone to use.
+
+When one of the multisite CRs (realm, zone group, zone) is deleted the underlying ceph realm/zone group/zone is not deleted, neither are the pools created by the zone. This must be done manually (see next section).
 
 For more information on the multisite CRDs please read [ceph-object-multisite-crd](ceph-object-multisite-crd.md).
 
+# Pulling a Realm
+
+If an admin wants to sync data from another cluster, the admin needs to pull a realm on a Rook Ceph cluster from another Rook Ceph (or Ceph) cluster. 
+
+To begin doing this, the admin needs 2 pieces of information:
+
+1. An endpoint in the realm you are pulling from. This endpoint must be from the master zone in that realm, and resolvable from the new Rook Ceph cluster.
+2. The access key and secret key of the system user from the realm you're pulling from. When you create a ceph-object-realm a system user gets created for the realm with an access key and a secret key. These keys are exported as a kubernetes [secret](https://kubernetes.io/docs/concepts/configuration/secret/) called "$REALM_NAME-keys" (ex: realm-a-keys).
+
+To get these keys from the cluster the realm was originally created on, run:
+```console
+$ kubectl -n $ORIGINAL_CLUSTER_NAMESPACE get secrets realm-a-keys -o yaml > ~/realm-a-keys.yaml 
```

Comment:
If it was a non-rook cluster, we won't be able to export it with kubectl. Seems like we need to document `ceph` commands that could either be run from the rook toolbox, or other ceph env. 
```suggestion
$ kubectl -n $ORIGINAL_CLUSTER_NAMESPACE get secrets realm-a-keys -o yaml > realm-a-keys.yaml 
```